styles/styles.css

```diff
@@ -132,16 +132,23 @@ html {
 }
 
 body {
-  font-size: var(--body-font-size-s);
+  font-size: var(--body-font-size-xs);
```

Comment:
on the Bitdefender site `body` default setup for `font-size` and `line-height` is: 
```
body,html {
    font-size: 1pc; //16px
    line-height: 1.5;
}
```
and should we not stick to the same setup?